### PR TITLE
feat(front end): link reason of play to backend (#78)

### PR DESCRIPTION
**Pull request by gargi-bansal**
### Purpose

Linked Play Reason to backend to create a chart on confirm, update reason in backend and then redirect to display cards.
[HRT-78](https://softeng761team5.atlassian.net/browse/HRT-78)

### Approach

Reason of play page now creates a new chart and updates it with the reason of play field.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

#### Does this PR involve a UI change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/pages/PlayReason.tsx [hunk 1]
```diff
@@ -84,6 +107,7 @@ const PlayReason: React.FC<RouteComponentProps> = (props) => {
               onChange={(e) => {
                 setState(e.target.value);
                 setError("");
+                console.log(e);
```
Comment (lucycy98) [inline src/pages/PlayReason.tsx]: can we remove this?

**Review by lucycy98 — DISMISSED** [review]

**Review by michellemtan — DISMISSED** [review]
LGTM! Works, tried it when playing first time and replaying 👍

File: src/pages/PlayReason.tsx (hunk 1 above)
Comment (gargi-bansal) [reply src/pages/PlayReason.tsx]: whoops, seems I left that in there by accident, it's fixed now @lucycy98

**Review by michellemtan — APPROVED** [review]

**Review by TCHE614 — APPROVED** [review]
LGTM!

**Pull request merged**